File: ReverseInteger/reverseInteger.js
/**
 * @param {number} x
 * @return {number}
 */
var reverse = function(x) {

  let temp = Math.abs(x);
  let temp_array = [];
  let isNegative = x < 0 ? true : false;
  let reverse_number;
  while (temp != 0) {
    temp_array.push(temp % 10);
    temp = Math.floor(temp / 10);
  }
  reverse_number = Number(temp_array.join(''));
  if (isNegative) {
    reverse_number *= -1;
  }
  return reverse_number;

};

const x = -1234;
let result;
result = reverse(x);
console.log(result);

File: ReverseInteger/reverseInteger.py
import math

class Solution:
    def reverse(self, x: int) -> int:
        n = abs(x)
        reversedNumString = "0"
        while n > 0:
            reversedNumString += str(n % 10)
            n = math.floor(n / 10)
        reversedNum = int(reversedNumString)
        # Validation only for instruction specification
        if reversedNum > 2147483647 or reversedNum < -2147483647:
            return 0
        if x < 0:
            return reversedNum * -1
        return reversedNum


x = 1534236469
sol = Solution()
print(sol.reverse(x))